packages/webdriver/src/command.ts

```diff
@@ -67,7 +77,7 @@ export default function (method, endpointUri, commandInfo, doubleEncodeVariables
             /**
              * inject url variables
              */
-            if (i < variables.length) {
+            if (variables.length) {
```

Comment:
didn't quite understand the logic we had before. Is the change intentional?

Reply:
will change it back